### DISCUSSION TITLE
Flashforge Firmware Tool - Windows binary

### DISCUSSION THREAD

**Discussion by GrandWaz00** [moonglow/FlashForge_Marlin#139] [General]
Original source: https://github.com/moonglow/flashforge_firmware_tool
Updated so that Dremel encryption key will be used if -k parameter is omitted on command line
Compiled for Windows

[flashforge_firmware_tool-Windows-DREMEL_default.zip](https://github.com/moonglow/FlashForge_Marlin/files/10580525/flashforge_firmware_tool-Windows-DREMEL_default.zip)

**Comment by 1Techpro** [comment]
I think this will work,  thanks again for your patience and support. I'll let you know how I make out.
